_config.yml

```diff
@@ -17,6 +17,9 @@ exclude:
 
 twitter_username: opensrcdesign
 github_username:  opensourcedesign
+mastodon:
+- username: @opensourcedesign
```

Comment:
@SaptakS should this part be defined as 'mastodon_username' instead do you think?

Reply:
I think that might be simpler for now to match it with the other PR. Though I like the idea of having both the instance and username configurable.